gensim/keras_integration/__init__.py

```diff
@@ -0,0 +1,8 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2011 Radim Rehurek <radimrehurek@seznam.cz>
+# Licensed under the GNU LGPL v2.1 - http://www.gnu.org/licenses/lgpl.html
+"""Keras wrappers for gensim.
```

Comment:
@tmylk The file `__init__.py` is common for the entire folder `gensim/keras_integration` (which in turn would have the files for integration of the various models with Keras). So shouldn't the docstring here be more generic (like it already is)?